cms/search/tests/test_kafka_integration.py

```diff
@@ -25,15 +25,15 @@ def setUpClass(cls):
         # so we can verify messages that come in.
         cls.consumer_created = KafkaConsumer(
             "search-content-updated",
-            bootstrap_servers=["localhost:9092"],
+            bootstrap_servers=settings.KAFKA_SERVER,
```

Comment:
q: does this not need a list?

Reply:
Per their docs: No, it can be a string or a list of strings. It just needs at least one broker that will respond to the meta data API request. But to keep it consistent with rest of the search work in the codebase, I updated this now to a list.